libavfilter/f_ebur128.c

```diff
@@ -696,6 +705,19 @@ static int filter_frame(AVFilterLink *inlink, AVFrame *insamples)
             /* override old cache entry with the new value */
             ebur128->i400.cache [ch][bin_id_400 ] = bin;
             ebur128->i3000.cache[ch][bin_id_3000] = bin;
+
+            ebur128->ch_raw_integrated_power[ch] = bin;
+        }
+
+        if (!ebur128->gate_measurement) {
```

Comment:
Can we move this code block closer to where we measure the gated integrated loudness?

Reply:
Need to add each sample to 'ungated_sum' and the 'ungated_count', so cann't move it.
But move the  part of  getting the average value and pan_law deduction
```
            if (!ebur128->gate_measurement) {
                ebur128->integrated_loudness = LOUDNESS(ebur128->ungated_sum / ebur128->ungated_count);
                /* dual-mono correction */
                if (nb_channels == 1 && ebur128->dual_mono) {
                    ebur128->integrated_loudness -= ebur128->pan_law;
                }
            }
```
closer to where calculate the gated integrated loudness.